contrib/IECoreAlembic/src/IECoreAlembic/AlembicScene.cpp

```diff
@@ -720,7 +976,7 @@ void AlembicScene::readTags( NameList &tags, int filter ) const
 
 void AlembicScene::writeTags( const NameList &tags )
 {
-	throw IECore::NotImplementedException( "AlembicScene::writeTags" );
+	IECore::msg( IECore::Msg::Warning, "AlembicScene::writeAttribute", "Not implemented" );
```

Comment:
> So Maya Sets don't export to Alembic at all?

Looking at the source for AbcExport, no, I don't think they do. Unless the Autodesk version is different?

> I don't think we'll be able to do much format comparison until we're at parity with the features

We couldn't reproduce a shot, for sure, but do I think we at least have enough to do some meaningful performance testing now.